src/apps/js_generic/js_generic.cpp

```diff
@@ -185,6 +186,51 @@ namespace ccfapp
     return JS_NewArrayBufferCopy(ctx, key.data(), key.size());
   }
 
+  static JSValue js_generate_rsa_key(
+    JSContext* ctx, JSValueConst, int argc, JSValueConst* argv)
+  {
+    if (argc != 1 && argc != 2)
+      return JS_ThrowTypeError(
+        ctx, "Passed %d arguments, but expected 1 or 2", argc);
+
+    int32_t key_size, key_exponent;
```

Comment:
Why not `uint32_t` together with `JS_ToUint32()`?

Reply:
Because I copied that part from `js_generate_aes_key` :-)